### DISCUSSION TITLE
Customizing actions in the relation manager modal

### DISCUSSION THREAD

**Discussion by james2doyle** [filamentphp/filament#14706] [Help] [answered]
### Package

Form builder

### Package Version

v3.2.119

### How can we help you?

I'm trying to customize the actions that are inside my relation manager modal.

![modal footer actions](https://github.com/user-attachments/assets/057a47eb-6cfe-4533-9d11-babe54d0036a)

I can't seem to figure out how to do that. I've tried using `getFormActions` on my `CreateRecord` page and that doesn't seem to work.

There also doesn't seem to be any methods on the RelationManager for changing the form actions - only the table ones.

I see a lot of info about the actions on tables but not a lot on the form actions when in a relation manager modal.

Anyone know how to customize these?

**Comment by leandrocfe** [comment, marked as answer]
Customize the CreateAction in the relation manager class

```php
$table
->headerActions([
    Tables\Actions\CreateAction::make()
        ->modalSubmitActionLabel('Create Record')
        ->modalCancelActionLabel('Abort')
        ->extraModalFooterActions([
            Action::make('newAction')
                ->action(function () {...}),
        ]),
])
```

**Reply by james2doyle** [reply]
Ok, thanks, this works! So editing the **tables** create action will edit the actions in the form modal footer? That is very confusing!
